.devcontainer/.dev_config.yaml

```
@@ -35,3 +35,4 @@ files_to_register_topic: internal_file_registry
 files_to_register_type: file_registered
 unstaged_download_event_topic: file_downloads
 unstaged_download_event_type: unstaged_download_requested
+auth_key: "placeholder"
```

```suggestion
auth_key: "{}"
```

Just to be consistent with the existing services and to indicate that it's a JSON object.